### ISSUE TITLE
Enable formatting to automatically apply to snippet generated code

### ISSUE THREAD

**Issue by sean-mcmanus** [microsoft/vscode-cpptools#5538]
It would be great if it just used the clang-format that was already set. Then you wouldn't even have to edit the snippets, but they would just conform to the existing code style you have chosen.

_Originally posted by @MrZeusTheCoder in https://github.com/microsoft/vscode-cpptools/issues/3556#issuecomment-631660899_

**Comment by elijahfhopp** [comment]
Ah, thank you. Should I have done that in the first place?

**Comment by sean-mcmanus** [comment]
@MrZeusTheCoder Yeah, usually it's best to keep different requests/issues separate, for tracking purposes.

**Comment by Dr-McFish** [comment]
hello, I understand that this is a planned feature. In the meantime, is there a manual way to change the formatting for C code snippets ?

**Comment by MrZeusTheCoder** [comment]
@Dr-McFish It appears to be in their "On Deck" project that has the description, "Issues that are prioritized to be fixed in some upcoming release, but haven't been committed to be fixed for a particular release." So it appears as though it is now a planned feature but is unimplemented.

You can always apply formatting afterward clang-format or such.

**Comment by Dr-McFish** [comment]
ok, thanks
